src/es5processor.ts

```diff
@@ -35,8 +35,8 @@ class ES5Processor extends Rewriter {
   unusedIndex: number = 0;
 
   constructor(
-      file: ts.SourceFile,
-      private pathToModuleName: (context: string, fileName: string) => string) {
+      file: ts.SourceFile, private pathToModuleName: (context: string, fileName: string) => string,
+      private isES5: boolean) {
```

Comment:
isES5 can be confusing given this class is called ES5 processor. Add some documentation to explain that this class sometimes consumes es6 code that still has commonjs module output, as a preprocessor for closure optimization.